CHANGELOG.mkd

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 Unreleased
 ----------
 
+- (RK-394) Fix dig path for --force for `puppetfile install` [#1269](https://github.com/puppetlabs/r10k/pull/1269)
```

Comment:
Actually, can you describe what this fixes? Doesn't it make it so you can override `force` from the CLI, where before you couldn't?

Reply:
Also might want to link the issue rather than the PR in this message.

Reply:
The [help](https://github.com/puppetlabs/r10k/blob/main/lib/r10k/cli/puppetfile.rb#L34) states that you can supply a `--force` flag, but it was looking in the wrong part of the hash map during initialization, so it wasn't working. I'm not sure what you are asking for, you want me to say `Make --force actually work correctly for puppetfile install`

Reply:
I like what you wrote 🙂